tests/embeddings/test_embeddings_multimodal_bridgetower_on_intel_cpu.sh

```diff
@@ -16,7 +16,7 @@ unset http_proxy
 function build_mmei_docker_images() {
```

Comment:
The scripts name should align with the microservice path.   
`intel_cpu` is the default device, don't need to add it.